docs/src/osdk/reference/commands/profile.md

```diff
@@ -0,0 +1,69 @@
+# cargo osdk profile
+
+## Overview
+
+The profile command is used to collect stack traces when running the target
```

Comment:
I am wondering if it is proper to call this command profile. As I understand, all it does it is to use GDB to collect the stack traces of a running kernel repeatedly. This is only one of the many means in performance profiling. And performance profiling may need to capture other metrics or data in order to identify bottlenecks. And ideally, a profiling tool should collect data, analyze the data, and eventually provide insights. This so-called "profile" command is too rudimentary to be advertised as a profiling tool or command.

So I think we have two suggestions.

The first option is to make the command an end-to-end profiling tool.

```bash
cargo osdk profile flame-graph [arguments]
```

The command will output the traces and a flame graph. In the future, we may add more tools to the profile command so that it can leverage all sorts of performance profiling techniques to help us identify bottlenecks easily.

The second option is to make the command more focus. 

```bash
cargo osdk sample_stacktraces
```

This command only collects stacktraces. This is exactly what the current implementation does. No more, no less.

Reply:
Want to postpone it.